hypothesis-python/RELEASE.rst

```diff
@@ -1,3 +1,3 @@
 RELEASE_TYPE: patch
 
-his patch refactors some internals to prepare for future work using our IR (:issue:`3921`).
+This patch refactors some internals to prepare for future work using our IR (:issue:`3921`).
```

Comment:
Given the flake, seems reasonable this time 🙂 

(if it was going to be more than a couple of hours faster, I'd prefer to re-run the flaky CI job and then fix the typo'd changelog in a later PR)